packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -30,400 +32,289 @@ export default {
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'disabled', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+
+export const HelpText = {
+  name: 'Help Text',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="Help text example"
+        placeholder="Placeholder"
+        rows="4"
+      ></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * Use the readonly attribute to render a textarea as readonly.  Interaction is enabled, but the textarea cannot be edited.  Events will be fired.
+ * Use the `placeholder` attribute to add a placeholder.
  */
-export const Readonly = {
-  parameters: {
-    controls: {
-      exclude: ['readonly']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'readonly', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+
+export const Placeholder = {
+  name: 'Placeholder',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="4"
+        spellcheck=""
+      ></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * This shows sd-textarea in its various sizes.
+ * Use the `rows` attribute to set the number of rows.
  */
 
-export const Sizes = {
-  parameters: {
-    controls: {
-      exclude: ['size']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Rows = {
+  name: 'Rows',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="4"
+        spellcheck=""
+      ></sd-textarea>
+
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="8"
+        spellcheck=""
+      ></sd-textarea>
+
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="12"
+        spellcheck=""
+      ></sd-textarea>
+    </div>
+  `
 };
 
```

Comment:
After Rows lets add "Label" section (see figma for reference) please.
Also please make sure the sections in code follow the same order as Figma.